src/components/views/rooms/MessageComposerInput.js

```diff
@@ -149,52 +132,56 @@ module.exports = React.createClass({
                     this.element.value = this.originalText;
                 }
 
-                self.resizeInput();
+                component.resizeInput();
                 return true;
             },
 
-            saveLastTextEntry: function() {
+            saveLastTextEntry: function () {
                 // save the currently entered text in order to restore it later.
                 // NB: This isn't 'originalText' because we want to restore
                 // sent history items too!
-                var text = this.element.value;
-                window.sessionStorage.setItem("input_" + this.roomId, text);
+                const contentHTML = contentStateToHTML(component.state.editorState.getCurrentContent());
+                console.error(contentHTML);
```

Comment:
Please clean up all of this - debug logging on `.error` is most infuriating if it leaks into the main codebase.